### ISSUE TITLE
Change string variable names

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/john-travoltage#100]
String variable names need to match a particular pattern.  Again, sorry I forgot to mention this!  The var should be the string key with dots removed, to camel case, with 'String' appended to the end of the name.

string key in john-travoltage-strings_en.json: scene.description
desired variable name: sceneDescriptionString

If you run the linting tools with ```grunt eslint```, it will give you an output of the string string vars that have incorrect names.

**Comment by jessegreenberg** [comment]
Thanks, closing.
